pkg/clusteragent/admission/mutate/auto_instrumentation.go

```diff
@@ -182,6 +181,26 @@ func injectLibConfig(pod *corev1.Pod, envKey string, envVal envValFunc) error {
 	return nil
 }
 
+// injectLibConfig injects additional library configuration extracted from pod annotations
+func injectLibConfig(pod *corev1.Pod, lang language) error {
+	configAnnotKey := fmt.Sprintf(common.LibConfigAnnotKeyFormat, lang)
+	confString, found := pod.GetAnnotations()[configAnnotKey]
+	if !found {
+		log.Debugf("Config annotation key %q not found on pod %s, skipping config injection", configAnnotKey, podString(pod))
+		return nil
+	}
+	log.Infof("Config annotation key %q found on pod %s, config: %q", configAnnotKey, podString(pod), confString)
+	var libConfig common.LibConfig
+	err := json.Unmarshal([]byte(confString), &libConfig)
```

Comment:
should we info log that we got a config and the version of config that we got?

Reply:
We're logging the full json string in line 192